sonar-plugin-api/src/main/java/org/sonar/api/batch/postjob/issue/PostJobIssue.java

```diff
@@ -62,24 +60,18 @@
   Integer line();
 
   /**
-   * Effort to fix the issue. Used by technical debt model.
-   */
-  @CheckForNull
-  Double effortToFix();
-
-  /**
    * Message of the issue.
    */
   @CheckForNull
   String message();
 
   /**
-   * Severity.
+   * Severity.  Only accurate in preview/issues mode.
```

Comment:
In fact I was wrong. PostJobIssues are only available in issues/preview mode so this attribute and isNew are always correct.